### ISSUE TITLE
can the options be changed on runtime ?

### ISSUE THREAD

**Issue by gchadid** [myloveCc/NETCore.MailKit#11]
The component works great staring on the startup.cs but i need to change de smtp setting on a controller, is that possible?

thanks

**Comment by anwarjaved** [comment]
Yes I need the same thing need to set settings according to Url, is that possible?
